index.d.ts

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
```

Comment:
I prefer `import {ReactNode, CSSProperties} from 'react'` here.

Reply:
Me too, but usually when using .d.ts definitions all i see is "import * as React ", i guess it should matter, changes made